pkg/operator/client/types/types.go

```diff
@@ -0,0 +1,48 @@
+package types
+
+// TODO: any way that this type can be removed? It has the same fields as HelmChart
+// OrderedDir represents a kots.io/v1beta1 HelmChart deployment
+type OrderedDir struct {
+	Name         string
+	Weight       int64
+	ChartName    string
+	ChartVersion string
+	ReleaseName  string
+	Namespace    string
+	UpgradeFlags []string
+}
```

Comment:
Left this note here since since the `OrderedDir` type mimics the `v1beta1.HelmChart` type, but one is used to install and the other is used to uninstall.